lib/adapter/freshheads/OutputAdapter.ts

```diff
@@ -31,10 +31,10 @@ export default class OutputAdapter implements Adapter {
 
             // Format of output bundle (uses hashes in non-development)
             // environment to bust browser caches
-            filename: isDev ? '[name].js' : '[name].[hash].js',
+            filename: isDev ? '[name].js' : '[name].[contenthash].js',
 
             // Format of output bundle chunks (when output was split up)
-            chunkFilename: isDev ? '[name].[id].js' : '[name].[hash].[id].js',
+            chunkFilename: isDev ? '[name].[id].js' : '[name].[contenthash].[id].js',
```

Comment:
Ik twijfel alleen wel of ik hier content hash mag gebruiken. Zie: https://webpack.js.org/configuration/output/#outputchunkfilename

In mijn test project zag ik zo snel geen problemen.

Reply:
Volgens deze site is dit wel de goede manier:
https://survivejs.com/webpack/optimizing/adding-hashes-to-filenames/

Reply:
Volgens mij mag je `contenthash` hier gewoon gebruiken. 

> By default `[id].js` is used or a value inferred from `output.filename`

output.filename verwijst naar https://webpack.js.org/configuration/output/#outputfilename. Als ik daar kijk zie ik `contenthash` tussen de opties staan.

Lijkt me dus helemaal prima!